src/main/kotlin/io/bazel/kotlin/plugin/jdeps/JdepsGenExtension.kt

```diff
@@ -70,9 +65,10 @@ import java.nio.file.Paths
  * @param configuration the current compilation configuration
  */
 class JdepsGenExtension(
-  val configuration: CompilerConfiguration,
-) :
-  AnalysisHandlerExtension, StorageComponentContainerContributor {
+  configuration: CompilerConfiguration,
+) : BaseJdepsGenExtension(configuration),
```

Comment:
Any reason this needs to be a base class and not just some independent class that both implementations can call into?

Reply:
either one works, but I figured I was going to use a common base for the 2 extensions anyways